Userland/Services/WebDriver/Client.cpp

```diff
@@ -34,6 +34,8 @@ Vector<Client::Route> Client::s_routes = {
     { HTTP::HttpRequest::Method::DELETE, { "session", ":session_id", "window" }, &Client::handle_close_window },
     { HTTP::HttpRequest::Method::POST, { "session", ":session_id", "element" }, &Client::handle_find_element },
     { HTTP::HttpRequest::Method::POST, { "session", ":session_id", "elements" }, &Client::handle_find_elements },
+    { HTTP::HttpRequest::Method::POST, { "session", ":session_id", "element", ":element_id", "element" }, &Client::handle_find_element_from_element },
+    { HTTP::HttpRequest::Method::POST, { "session", ":session_id", "element", ":element_id", "elements" }, &Client::handle_find_elements_from_element },
```

Comment:
`handle_find_elements_from_element` added in the wrong commit

Reply:
My git-fu is a bit rusty... But the git-fix-shit-fu is strong!